sea-query-driver/src/sqlx_sqlite.rs

```diff
@@ -29,10 +29,10 @@ pub fn bind_params_sqlx_sqlite(input: TokenStream) -> TokenStream {
 
     let with_time = if cfg!(feature = "with-time") {
         quote! {
-            v @ Value::TimeDate(_) => query.bind(v.time_as_naive_utc_in_string()),
-            v @ Value::TimeTime(_) => query.bind(v.time_as_naive_utc_in_string()),
-            v @ Value::TimeDateTime(_) => query.bind(v.time_as_naive_utc_in_string()),
-            v @ Value::TimeDateTimeWithTimeZone(_) => query.bind(v.time_as_naive_utc_in_string()),
```

Comment:
`time_as_naive_utc_in_string` is still used by `rusqlite` and cannot be removed.